roxctl/central/generate/k8s.go

```diff
@@ -80,7 +80,6 @@ func orchestratorCommand(shortName, _ string) *cobra.Command {
 	}
 	if !roxctl.InMainImage() {
```

Comment:
How about moving the conditions from OutputZip here then we will have a valid config and only need to check if output dir is set or not. No need to pass flags everywhere.
What do you think?

Reply:
I suppose that is an option as long as there are no unintended side effects. Behavior would be different in the following case:

If `InMainImage()` is true and we also set an `--output-dir`, the current implementation would output a zip to `STDOUT` instead of a file. Changing this condition would change the behavior in this edge case to output a file instead. But I suppose that is actually correct, and it's unlikely that we currently have such a case. I'll search for it anyways and then try this.